concertOrganizer/src/com/codecool/emis/concertorganizer/band/Band.java

```diff
@@ -0,0 +1,49 @@
+package com.codecool.emis.concertorganizer.band;
+
+import com.codecool.emis.concertorganizer.ConcertOrganizer;
+import com.codecool.emis.concertorganizer.concert.Concert;
+
+import java.util.Set;
+
+public class Band {
+
+    private String name;
+    private Style sytle;
+    private ConcertOrganizer concertOrganizer;
+
+    public Band(String name, Style sytle, ConcertOrganizer concertOrganizer) {
+        this.name = name;
+        this.sytle = sytle;
+        this.concertOrganizer = concertOrganizer;
+    }
+
+    public Style getStyle() {
+        return sytle;
+    }
+
+
+    public String getName() {
+        return name;
+    }
+
+
+    public void choiceWarmUpBand(Band band){
```

Comment:
usually we would do these kind of logic in the aggregator class, because it's the concert organizer who has all the necessary data. 
in this case it's okay,because the exercise asked you to do it from the band class, but if it wasn't the case you would do it in the ConcertOrganizer class without having a concertOrganizer field here
in this exercise this is fine, I just wanted you to know